setup.ts

```diff
@@ -153,9 +153,21 @@ async function superAdmin(): Promise<void> {
   }
 }
 
+// Function to check if local MongoDB instance is running
+async function checkLocalMongoDB(): Promise<boolean> {
+  try {
+    await mongodb.connect("mongodb://localhost:27017", {
```

Comment:
Why have you hard coded localhost? Is this only implemented when there is no MONGO_DB_URL configured?

Reply:
This only checks if a local instance of mongo is running or not, do you want setup.ts to also check if the user has added any other mongo url to .env before running the "npm run setup" command?

Reply:
If there is already a mongo URL in the .env file, then that should be used as the value for the prompt. Use this as as reference.
- https://github.com/PalisadoesFoundation/talawa-api/pull/1632

Reply:
Done, now the setup.ts also checks for any mongourl in already present in .env